mne/minimum_norm/inverse.py

```diff
@@ -1294,6 +1293,9 @@ def make_inverse_operator(info, forward, noise_cov, loose='auto', depth=0.8,
         detected automatically. If int, the rank is specified for the MEG
         channels. A dictionary with entries 'eeg' and/or 'meg' can be used
         to specify the rank for each modality.
+    use_cps : None | bool (default None)
+        Whether to use cortical patch statistics to define normal
+        orientations. Only used when surf_ori and/or force_fixed are True.
```

Comment:
remove

Only used when surf_ori and/or force_fixed are True.

it's not params of this function

Reply:
But you actually need it here. When doing depth weighting, you must provide a free orientation forward. So conversion happens later, within `make_inverse_operator`.

Reply:
(and, as you say, these conversions are now sometimes automatic, so whether or not CPS should be used needs to be a param in those cases, as well)

Reply:
what I mean is that make_inverse_operator does not have surf_ori or force_fixed parameters.
So the docstring is unclear.

Reply:
Ahh I see, hopefully fixed in second commit below